src/App.css

```diff
@@ -1361,25 +1356,22 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
     padding-top: 10px;
     width: var(--mobile-view-width);
     left: 0%;
-    height: 35%;
-    min-height: 340px;
+    height: 40%;
     top: 0;
-    display: flex;
-    flex-direction: column;
-    align-self: start;
-    justify-content: flex-start;
```

Comment:
I pretty much returned all of the popup properties back to what they were on master and made adjustments around it.